packages/select/stories/select.stories.tsx

```diff
@@ -10,16 +10,12 @@ import Select, { SelectField, SelectFieldProps } from '../src';
 import FormikResults from '../../../story-utils/FormikResults';
 import { singleValueSchema, multiValueSchema, options, SelectedOption, autofillOptions } from './utils';
 
+/**
+ * Select dropdown without a Label or Feedback
```

Comment:
this should be on the select story level or the selectfield should be split into it's own stories file

Reply:
oh that makes sense